src/constants.ts

```diff
@@ -48,7 +50,7 @@ export const PUBLIC_NETWORKS: { [chainId: number]: { name: string; etherscan: st
 export const DEFAULT_BLOCKCHAIN_EXPLORER_DOMAIN = "https://etherscan.io";
 
 export const DEFAULT_CACHING_TTL = 60 * 60 * 24 * 7 * 2; // 2 Weeks
-export const DEFAULT_CACHING_SAFE_LAG = 60 * 60 * 24; // 1 Day
+export const DEFAULT_CACHING_SAFE_LAG = Number(process.env.CACHING_SAFE_LAG) ?? 60 * 10; // 10 mins
```

Comment:
I'd prefer to default conservative, and opt-in via overriding. wdyt about setting it at something like 1 hour, and then we can crank it up as needed?

Also, I'm not super keen on having this read directly from the environment, but I don't have an alternate proposal and I don't think we should let that hold this up, but it would be good to handle it in another way in future.

Reply:
ok i'm fine with that, defaulted back to 1 hour. In the relayer-v2 repo this would be set via Config. I can make it an input var